config/promitor/resource-discovery/resource-discovery-declaration.yaml

```diff
@@ -75,6 +75,8 @@ resourceDiscoveryGroups:
   type: SynapseSqlPool
 - name: synapse-workspaces
   type: SynapseWorkspace
+- name: traffic-managers
```

Comment:
Can you open a PR to https://github.com/promitor/testing-infrastructure with a test resource please?

Reply:
Here it is - https://github.com/promitor/azure-infrastructure/pull/48